packs/data/equipment.db/lastwall-soup-improved.json

```
@@ -0,0 +1,111 @@
+{
+    "_id": "VcEd9Bh4q5vQfpLv",
+    "data": {
+        "activation": {
+            "condition": "",
+            "cost": 0,
+            "type": ""
+        },
+        "autoDestroy": {
+            "_deprecated": true,
+            "value": true
+        },
+        "autoUse": {
+            "_deprecated": true,
+            "value": true
+        },
+        "baseItem": null,
+        "charges": {
+            "_deprecated": true,
+            "max": 1,
+            "value": 1
+        },
+        "consumableType": {
+            "value": "elixir"
+        },
+        "consume": {
+            "_deprecated": true,
+            "value": ""
+        },
+        "containerId": null,
+        "description": {
+            "value": "<p><strong>Activate</strong> 1 minute (Interact)</p>\n<hr />\n<p>Crimson Reclaimers make this rich, hearty soup using herbs that baffle the senses of the undead. A bowl of Lastwall soup is as nourishing as a full meal. Gravelands Herbalists have spent time with the Knights of Lastwall, and have learned how to improve upon Lastwall soup.</p>\n<p>For 1 hour after consuming a bowl of improved Lastwall soup, you gain a +3 item bonus to Stealth checks and DCs against undead.</p>\n<hr />\n<p>@Compendium[pf2e.consumable-effects.Effect: Lastwall Soup (Improved)]{Effect: Lastwall Soup (Improved)}</p>\n<hr />\n<p><em>Note: This item can only be made by characters with the @Compendium[pf2e.feats-srd.Gravelands Herbalist]{Gravelands Herbalist} feat. Additionally, Paizo did not provide a price, so we set it as appropriate for a level 10 consumable.</em></p>"
```

```suggestion
            "value": "<p><strong>Activate</strong> 1 minute (Interact)</p>\n<hr />\n<p>Crimson Reclaimers make this rich, hearty soup using herbs that baffle the senses of the undead. A bowl of Lastwall soup is as nourishing as a full meal. Gravelands Herbalists have spent time with the Knights of Lastwall, and have learned how to improve upon Lastwall soup.</p>\n<p>For 1 hour after consuming a bowl of improved Lastwall soup, you gain a +3 item bonus to Stealth checks and DCs against undead.</p>\n<hr />\n<p>@Compendium[pf2e.consumable-effects.Effect: Lastwall Soup (Improved)]{Effect: Lastwall Soup (Improved)}</p>\n<hr />\n<p><em>Note:  Paizo did not provide a price for this item, so we set it as appropriate for a level 10 consumable.</em></p>"
```
We don't want to reiterate rule info in our editorial notes.